File: atsa/src/main/java/jp/kaiz/atsassistmod/ifttt/IftttEditView.java
// SPDX-License-Identifier: LGPL-3.0-or-later
// Copyright © 2026 mirukuneko and RealTrainModRenewed contributors
package jp.kaiz.atsassistmod.ifttt;

/** Decouples IFTTT container types from the concrete editor screen (was GUIIFTTTMaterial). */
public interface IftttEditView {
    String getTextFieldText(int index);
    int getTextFieldInt(int index);
    int textFieldLength();
}

File: atsa/src/main/kotlin/jp/kaiz/atsassistmod/ifttt/IftttEditView.kt
// SPDX-License-Identifier: LGPL-3.0-or-later
// Copyright © 2026 mirukuneko and RealTrainModRenewed contributors
package jp.kaiz.atsassistmod.ifttt

/** Decouples IFTTT container types from the concrete editor screen. */
interface IftttEditView {
    fun getTextFieldText(index: Int): String

    fun getTextFieldInt(index: Int): Int

    fun textFieldLength(): Int
}
